src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -851,6 +910,7 @@ def _visit_computation_node(self, node: ast.With) -> List[gt_ir.ComputationBlock
         # Parse computation specification, i.e. `withItems` nodes
         iteration_order = None
         interval = None
+        parallel_intervals = [None]
```

Comment:
For computations without a parallel interval specification there should still be parallel intervals, but from `-inf` to `inf` right?